geniza/corpus/models.py

```diff
@@ -676,6 +686,13 @@ def iiif_images(self, filter_side=False):
                             "excluded": b.side and i not in b.selected_images,
                         }
 
+        if with_placeholders:
```

Comment:
need a comment here for context
```suggestion
        # when requested, include any placeholder canvas URIs referenced by any associated transcriptions
        if with_placeholders:
```